packages/k2-alpine/src/components/Chip/Chip.tsx

```diff
@@ -11,79 +11,96 @@ import { useTheme } from '../../hooks'
 import { Icons } from '../../theme/tokens/Icons'
 import { alpha } from '../../utils'
 
-export type ChipSize = 'small' | 'large'
-
+type ChipSize = 'small' | 'large'
+type ChipVariant = 'light' | 'dark'
 type ChipIconType = 'expandMore'
 
 interface ChipProps {
   onPress?: () => void
   style?: StyleProp<ViewStyle>
   testID?: string
   size: ChipSize
+  variant?: ChipVariant
   rightIcon?: ChipIconType | JSX.Element
   hitSlop?: number | Insets
 }
 
 export const Chip = forwardRef<
   React.ElementRef<typeof TouchableOpacity>,
   ChipProps & PropsWithChildren
->(({ size, rightIcon, style, children, testID, ...rest }, ref) => {
-  const { theme } = useTheme()
+>(
+  (
+    { size, variant = 'light', rightIcon, style, children, testID, ...rest },
+    ref
+  ) => {
+    const { theme } = useTheme()
 
-  const tintColor = theme.colors.$textPrimary
-  const backgroundColor = theme.colors.$surfaceSecondary
-  const iconWidth = { large: 20, small: 16 }[size]
+    const tintColor = {
+      light: theme.colors.$textPrimary,
+      dark: theme.isDark
+        ? theme.colors.$textPrimary
+        : theme.colors.$surfacePrimary
+    }[variant]
+    const backgroundColor = {
+      light: theme.colors.$surfaceSecondary,
+      dark: alpha('#58585B', 0.8)
```

Comment:
58585B is neutral700 in k2-mobile (the old k2). can you ask design if we can use this as neutral700 in k2-alpine. if yes, let's put it in the theme.